homework/01LessonOne.js

```diff
@@ -144,56 +144,59 @@ var notNotUndefined;
 // Explain truthiness and falsiness in your own words. Provide an example for each.
 //
 // Your Answer Goes Here:
+//Something is considered truthy when it is considered to be true and trutiness is the extent of its truth
+// i.e.  '4' = 4 is true  but '4' === 4 is false because they are not of true to the same extent
```

Comment:
This is a good start. Further reading.

https://developer.mozilla.org/en-US/docs/Glossary/Truthy
https://developer.mozilla.org/en-US/docs/Glossary/Falsy